### PR TITLE
omit invalid HTML props from the rendered div tag that have been pass…

### LINKED ISSUE #34: Invalid Props on div element
When using the component React fires an 

`Unknown props `callback`, `expiredCallback`, `locale` on <div> tag. Remove these props from the element.`

Warning. This is due to the way that the component props are passed down to the actual div element without stripping the purely functional component props.

### PR DESCRIPTION
**Pull request by fde31**
…ed the component's props until now

fixes #34

### CODE REVIEW

**Comment by fde31** [comment]
seems to be related to linting issues. However the linting setup seems a bit messed up or am I missing something?! I switched from Node v6.6.0 to Node v4.3.1 in order to get the peerDeps right (maybe worth updating the repo for that, too!?) but still no luck. Punting on this for now as I assume that it might be quicker and easier for u to resolve that!? Otherwise I'm happy to help. Just lemme know.

**Comment by evenchange4** [comment]
For eslint dependency problem please try to install `npm@2`, I create an issue #36 for this problem.

File: src/Recaptcha.js [hunk 1]
```diff
@@ -50,9 +51,10 @@ class Recaptcha extends React.Component {
   render() {
     const { className, sitekey, ...otherProps } = this.props;
 
+
     return (
       <div
-        {...otherProps}
+        {...omit(otherProps, Object.keys(this.constructor.propTypes)) }
```
Comment (evenchange4) [inline src/Recaptcha.js]: I think there will be a problem if consumers remove `propTypes` in their project by [babel-plugin-transform-react-remove-prop-types](https://github.com/oliviertassinari/babel-plugin-transform-react-remove-prop-types). (Actually I use this babel-plugin in all my projects.)

How about  this:

```
{...omit(otherProps, ['props', 'to', 'omit']) }
```

**Comment by fde31** [comment]
ah sure. we could do that. didn't think about the propTypes removal. so I guess hardcoding the props might be the way to go. will take care

**Comment by evenchange4** [comment]
Thanks! Released `v1.1.1` with your changes.

**Pull request merged**